### PR TITLE
Use native reponav dropdown if available

### LINKED ISSUE #1269: Detect custom GHE tabs duplicating custom More tab
left release

right release

When using native custom tab, `refined-github` could use the already present one.

### PR DESCRIPTION
**Pull request by bfred-it**
Fixes #1269 

Not tested on GHE. @Naramsim can you try this branch?

Style matches the native one as well:

And it should look like this on GHE (mockup): 

ghe mockup

### CODE REVIEW

File: source/features/more-dropdown.js [hunk 1]
```diff
@@ -6,31 +6,47 @@ import {appendBefore} from '../libs/utils';
...
+}
+
+export default function () {
+	getDropdown().append(
+		<a href={`/${repoUrl}/compare`} class="dropdown-item" data-skip-pjax>
+			{icons.darkCompare()}
+			{' Compare'}
+		</a>,
+
+		pageDetect.isEnterprise() ? '' :
```
Comment (bfred-it) [inline source/features/more-dropdown.js]: @busches can we drop `isEnterprise` from here now? (Blaming #739)

**Comment by Naramsim** [comment]
moreok

Fine on my side 😄  You are so quick!

**Comment by bfred-it** [comment]
Since you're here, do you have a Dependencies page on your GHE? Likehttps://github.com/sindresorhus/refined-github/network/dependencies

File: source/features/more-dropdown.js (hunk 1 above)
Comment (busches) [reply source/features/more-dropdown.js]: I doubt it'll ever be added, in my case GHE is self-hosted and behind a firewall and not likely able to pull the information it needs to populate that information.

**Pull request merged**